src/main/java/org/gridsuite/network/map/NetworkMapService.java

```diff
@@ -165,6 +169,13 @@ private static TwoWindingsTransformerMapData toMapData(TwoWindingsTransformer tr
         return builder.build();
     }
 
+    private static TapChangerData toMapData(TapChanger<?, ?> phaseTapChanger) {
```

Comment:
Rename parameter phaseTapChanger to tapChanger (because it can be a ratio tap changer or a phase tap changer)